gramex/__main__.py

```diff
@@ -10,3 +10,4 @@
 import gramex
 
 gramex.commandline()
+
```

Comment:
Can we remove whitespace-only changes?

Reply:
Done. My bad, should have sanitized beforehand.